src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,76 @@
+package calculator;
+
+public class StringCalculator {
+    private static final int SPLIT_BOUNDARY = 2;
+    private static final int SEPARATOR_LENGTH = 1;
+    private static final int SEPARATOR = 1;
+
+    private static final String AFTER_SEPARATOR = "//";
+
+    private CustomSeparatorGroup customSeparatorGroup;
+
+    StringCalculator() {
+        customSeparatorGroup = new CustomSeparatorGroup();
+    }
+
+    public CustomSeparator createCustomSeparator(String input) {
+        String[] unrefinedSeparator = input.split(AFTER_SEPARATOR);
+        isValidSplitInput(unrefinedSeparator.length, SPLIT_BOUNDARY);
+        isValidSplitInput(unrefinedSeparator[SEPARATOR].length(), SEPARATOR_LENGTH);
+
+        return new CustomSeparator(unrefinedSeparator[SEPARATOR]);
+    }
+
+    private void isValidSplitInput(int length, int splitBoundary) {
+        if (length != splitBoundary) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    public int[] splitBySeparator(String expression) {
+        String[] temp = expression.split(customSeparatorGroup.combineSeparatorToRegex());
+        int[] result = new int[temp.length];
```

Comment:
```suggestion
        String[] numbers = expression.split(customSeparatorGroup.combineSeparatorToRegex());
        return Arrays.stream(numbers)
                .mapToInt(Integer::parseInt)
                .toArray();
```
stream을 사용하면 안전하고 읽기 좋은 코드로 만들 수 있습니다
배열보다 컬렉션을 사용하도록 작성해보아요

Reply:
넵 감사합니다 ~